docs/index.html

```diff
@@ -152,9 +162,7 @@
 {% include 'partials/component/masthead.njk' %}
 {% include 'partials/component/sidenav.njk' %}
 
-<rh-surface id="main"
-            role="main"
-            color-palette="lightest">
+<rh-surface id="main" role="main">
```

Comment:
same as previous, remove rh-surface in preference of `<main>`

Reply:
81b1e9f0